### ISSUE TITLE
it cost long time for atom3d/examples/ppi/gnn /data.py

### ISSUE THREAD

**Issue by bozhenhhu** [drorlab/atom3d#64]
when run the train.py for ppi/gnn, it spends a lot of time to load data in "for graph1, graph2 in dataloader:", but nothing outputs.

if __name__=="__main__":
    from tqdm import tqdm
        
    dataset = PPIDataset(os.path.join('/usr/data/data/DIPS-split/data/', 'train'))
    dataloader = DataLoader(dataset, batch_size=3, shuffle=False, collate_fn=CollaterPPI(batch_size=3), num_workers=4)
    for graph1, graph2 in dataloader:
        print(graph1)
        print(graph2)
        break

is there problems?
